### PR TITLE
test: fixed non deterministic result check

### PR DESCRIPTION
**Pull request by fmartingr**
#### Summary

Fixed a test that was expecting deterministic results.

#### Ticket Link

NONE

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by gabrieljackson — APPROVED** [review]
👍

**Comment by mirshahriar** [comment]
I think this equal check doesn't consider order.

**Comment by fmartingr** [comment]
> I think this equal check doesn't consider order.

I made some tests and [it does](https://go.dev/play/p/VYB1I1qSXwq), that's why they have `assert.Subset` for this scenarios.

**Comment by mirshahriar** [comment]
> 

You are right. Could you please also check the length. Subset checks may pass while size is not same. `assert.Subset(t, [1, 2, 3], [1, 2]) is true`

**Comment by fmartingr** [comment]
> > 
> 
> You are right. Could you please also check the length. Subset checks may pass while size is not same. `assert.Subset(t, [1, 2, 3], [1, 2]) is true`

You're totally right. 👍

File: internal/tools/aws/tags_test.go [hunk 1]
```diff
@@ -53,8 +57,9 @@ func TestTagsASRDSTags(t *testing.T) {
 	key2 := "key2"
 	value2 := "value2"
 	tags, err := NewTags(key, value, key2, value2)
+	result := tags.ToRDSTags()
 	assert.NoError(t, err)
```
Comment (mirshahriar) [inline internal/tools/aws/tags_test.go:61]: It is preferable to check error immediately after it occurs.

In this case, just after
```
tags, err := NewTags(key, value, key2, value2)
```

**Review by mirshahriar — APPROVED** [review]

**Pull request merged**